py/selenium/webdriver/remote/webdriver.py

```diff
@@ -1017,6 +1020,32 @@ def get_log(self, log_type):
         """
         return self.execute(Command.GET_LOG, {"type": log_type})["value"]
 
+    def start_devtools(self):
```

Comment:
I'm not sure about the naming here, any suggestions are welcome.

Reply:
Java calls this `getDevTools()`, I am not sure what is the idiomatic fit for Python.

Reply:
to start a devtools session in Java ->  devTools.createSession();

Reply:
This method both _gets_ DevTools (loads the proper version) and _creates session_, then returns both the loaded devtools module and the established connection. `start_devtools` is the best possible name I came up with for this 2 operations. We could of course split them into 2 separate methods...

Reply:
for bidi this should be an implementation detail in the higher level APIs, so not too concerned about it.